en/micro-integrator/docs/references/connectors/salesforce-rest-connector/sf-rest-connector-example.md

```diff
@@ -56,52 +56,48 @@ Follow these steps to set up the ESB Solution Project and the Connector Exporter
+        3. To get the input values in to the API we can use the [property mediator](https://ei.docs.wso2.com/en/next/micro-integrator/references/mediators/property-Mediator/) . Navigate into the **Palette** pane and select the graphical mediators icons listed under **Mediators** section. Then drag and drop the `Property` mediators into the Design panel as shown bellow.
     
             <img src="../../../../assets/img/connectors/salesforce-api-drag-and-drop-property-mediator.png" title="Add property mediators" width="800" alt="Add property mediators"/>
 
             The parameters available for configuring the Property mediator are as follows:
     
             > **Note**: That the properties should be add to the pallet before create the operation.
     
-        5. Add the property mediator to capture the `sObjectName` value.The sObjectName type can be used to retrieve the metadata for the Account object using the GET method, or create a new Account object using the POST method. In this example we are going to create a new Account object using the POST method.
+        4. Add the property mediator to capture the `sObjectName` value.The sObjectName type can be used to retrieve the metadata for the Account object using the GET method, or create a new Account object using the POST method. In this example we are going to create a new Account object using the POST method.
    
             - **name** : sObjectName
             - **expression** : json-eval($.sObject)
             - **type** : STRING
    
             <img src="../../../../assets/img/connectors/salesforce-api-property-mediator-values1.png" title="Add values to capture sObjectName value" width="800" alt="Add values to capture sObjectName value"/>
     
-        6. Add the property mediator to capture the `fieldAndValue` values. The fieldAndValue contains object fields and values that user need to store.
+        5. Add the property mediator to capture the `fieldAndValue` values. The fieldAndValue contains object fields and values that user need to store.
    
             - **name** : fieldAndValue
             - **expression** : json-eval($.fieldAndValue)
             - **type** : STRING
      
             <img src="../../../../assets/img/connectors/salesforce-api-property-mediator-values2.png" title="Add values to capture fieldAndValue value" width="800" alt="Add values to capture fieldAndValue value"/>  
     
-3. Creating the sequence to retrive the salesforce objects created.
+3. Creating the sequence to retrive the created salesforce objects.
 
+   For this you need to define a new sequence called `retrive`. Please follow the steps given in 2 for create a new sequence.
+    
     1. Initialize the connector.
     
         You can use the generated tokens to initialize the connector. Please follow the steps  given in 2.1 for setting up the `init` operation to the `retrive.xml` sequence. 
```

Comment:
Change the numbering. I guess this will be "2.a". Please check and update in all the sections.

Reply:
updated.